src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -854,6 +855,16 @@ namespace
             QVBoxLayout* layout = new QVBoxLayout();
             groupbox->setLayout(layout);
 
+            QFormLayout* sublayout = create_form_layout();
+            layout->addLayout(sublayout);
+
+            QComboBox* light_sampling;
+            light_sampling = create_combobox("advanced.algorithm");
```

Comment:
We want to give a more descriptive name to that setting, I suggest `advanced.light_sampler.algorithm`. 

You'll also need to change the `create_direct_link()` call accordingly:

    create_direct_link("advanced.light_sampler.algorithm", "light_sampler.algorithm");